docker/intel-dgpu-driver.Dockerfile

```diff
@@ -34,6 +34,15 @@ RUN git clone -b ${I915_RELEASE} --single-branch https://github.com/intel-gpu/in
     && cp defconfigs/i915 .config \
     && make olddefconfig && make modules -j $(nproc) && make modules_install
 
+# Copy out-of-tree drivers to /opt/lib/modules/${KERNEL_FULL_VERSION}/ and strip debug symbols from modules
+RUN for file in $(find /lib/modules/${KERNEL_FULL_VERSION}/updates/ -name "*.ko"); do \
+    dname=$(dirname /opt$file); mkdir -p $dname; cp $file /opt$file; strip --strip-debug /opt$file; done
+
+# Copy in-tree driver dependencies to /opt/lib/modules/${KERNEL_FULL_VERSION}/
+ARG MODULES="video sysimgblt sysfillrect syscopyarea fb_sys_fops i2c-algo-bit drm rc-core cec drm_kms_helper drm_display_helper"
+    RUN for file in $(for mod in $MODULES; do find /lib/modules/${KERNEL_FULL_VERSION}/ -name $mod.ko.xz; done); do \
+        dname=$(dirname /opt$file); mkdir -p $dname; cp $file /opt$file; done
+
```

Comment:
I prefer not to use this solution, manually picking up and packaging the dependant in-tree module might not the best option for us. Even it reduces the package size, it might also increase the risk for users to miss some in-tree modules dependency. Besides, It also increases maintaining efforts when we update the OOT modules.
I prefer to wait for the final solution in KMM and use the in-tree kernel modules on the host.

Reply:
Got it, sounds good.